tests/testing/LocalBearTestHelperTest.py

```diff
@@ -241,3 +244,23 @@ def test_timeout_deprecation_warning(self):
                               invalid_files=files, timeout=50)
             self.assertIn('timeout is ignored as the timeout set in the repo '
                           'configuration will be sufficient', stderr.getvalue())
+
+
+def mock_test(self):
+    return os.getcwd()
+
+class ChangeBaseDirectoryTest(unittest.TestCase):
+    def setUp(self):
+        self.old_cwd = os.getcwd()
+
+    @patch(__name__
+           + '.' + 'verify_local_bear.LocalBearTest.test_valid_files.test',
+           side_effect=mock_test)
+    def test_changedirectory(self, test):
+        new_cwd = verify_local_bear(TestBear,
+                                    valid_files=files,
+                                    invalid_files=(),
+                                    base_directory='tests/bears',
+                                    settings={'result': True},
+                                    )
+        self.assertNotEqual(self.old_cwd, new_cwd)
```

Comment:
You probably want to mock the `change_directory` method and see if it's called with `tests/bears` here. Additionally, you can check cwd == 'tests/bears' explicitly but that might be unnecessary since `change_directory` is most likely already being tested somewhere.